pkgs/development/compilers/openjdk/default.nix

```
@@ -1,32 +1,32 @@
 { stdenv, lib, fetchurl, bash, pkgconfig, autoconf, cpio, file, which, unzip
 , zip, perl, cups, freetype, alsaLib, libjpeg, giflib, libpng, zlib, lcms2
 , libX11, libICE, libXrender, libXext, libXt, libXtst, libXi, libXinerama
-, libXcursor, libXrandr, fontconfig, openjdk11
+, libXcursor, libXrandr, fontconfig, openjdk13-bootstrap
 , setJavaClassPath
 , headless ? false
 , enableJavaFX ? openjfx.meta.available, openjfx
 , enableGnome2 ? true, gtk3, gnome_vfs, glib, GConf
 }:
 
 let
-  major = "12";
+  major = "13";
   update = ".0.2";
-  build = "ga";
+  build = "-ga";
 
   openjdk = stdenv.mkDerivation rec {
     pname = "openjdk" + lib.optionalString headless "-headless";
-    version = "${major}${update}-${build}";
+    version = "${major}${update}${build}";
```

to be consistent with the previous expressions
```suggestion
    version = "${major}${update}-${build}";
```

---

I did it this way because when I initially started working on this, the initial release of 13 was "13+33", i.e. there was no "-" character anywhere in the version.

---

ah, interesting. Disregard then